R/cv_varsel.R

```diff
@@ -609,26 +609,24 @@ loo_varsel <- function(refmodel, method, nterms_max, ndraws,
   # turn, i.e., brms's `reloo` argument) currently cannot be used because all
   # these techniques result in new MCMC draws for the reference model, meaning
   # that the projection would have to be adapted. Therefore, it is easier to
-  # recommend K-fold CV (for the reference model refits, i.e., brms's `reloo`
+  # recommend K-fold-CV (for the reference model refits, i.e., brms's `reloo`
   # argument, another reason is that they can quickly become as costly as
-  # K-fold CV).
-  warn_pareto(
-    n07 = sum(pareto_k > 0.7), n05 = sum(0.7 >= pareto_k & pareto_k > 0.5),
-    warn_txt_start = paste0("In the calculation of the reference model's ",
-                            "PSIS-LOO CV weights, "),
-    warn_txt_mid_common = paste0(" (out of ", n, ") Pareto k-values are "),
-    warn_txt_end = paste0(
-      ". Moment matching (see the loo package), mixture importance ",
-      "sampling (see the loo package), and `reloo`-ing (see the brms package) ",
+  # K-fold-CV).
+  warn_pareto(n07 = sum(pareto_k > 0.7), n = n,
+              khat_threshold = .ps_khat_threshold(dim(psisloo)[1]),
+    warn_txt = paste0("Some Pareto k's for the reference model's ",
+      "PSIS-LOO weights are >%s (%d / %d).",
```

Comment:
I would prefer moving ` (%d / %d)` after `"Some` at the beginning of the message because otherwise, unexperienced users might think the ratio relates to the `khat_threshold` printed by `%s`.

Reply:
At that occasion, do we perhaps also want to add a short explanation where the value printed by `%s` comes from? Often, 0.7 should be printed (which is probably established enough that it gets quickly recognized by many users), but given that the recommendation of a Monte-Carlo-sample-size-specific k-hat threshold is quite recent, many users might not be aware where a value other than 0.7 (depending on the Monte Carlo sample size) comes from. I'm thinking of adding something like ` (the k-hat threshold for the given Monte Carlo sample size)` after `%s` (this would be possible if we moved the ` (%d / %d)` parenthesis up front, as suggested in my previous comment).

Reply:
I moved `(%d / %d)`. I made the warning message even shorter and improved the `cv_varsel()` doc.

Reply:
Thanks!